.coveragerc

```diff
@@ -624,7 +624,6 @@ omit =
     homeassistant/components/notify/telstra.py
     homeassistant/components/notify/twitter.py
     homeassistant/components/notify/xmpp.py
-    homeassistant/components/notify/yessssms.py
```

Comment:
Before we remove this from `.coveragerc`, please add some tests for the remaining lines that are missed here:

https://coveralls.io/builds/19314192/source?filename=homeassistant%2Fcomponents%2Fnotify%2Fyessssms.py#L58

Reply:
think that looks better 
https://coveralls.io/builds/19333051/source?filename=homeassistant/components/notify/yessssms.py

Reply:
Looks great!